test/expected.py

```diff
@@ -23,6 +23,176 @@
+    (u'http://localtest.me:8000/test_pages/shared/test_script_2.js',
+        u'http://localtest.me:8000/test_pages/http_test_page_2.html',
+        u'')
+}
+
+http_cached_requests = {
+    (u'http://localtest.me:8000/test_pages/http_test_page.html',
+        None,
+        0, 0, 1, None, None,
+        u'[System Principal]',
+        u'chrome://browser/content/browser.xul', 6),
+    (u'http://localtest.me:8000/test_pages/shared/test_script_2.js',
+        u'http://localtest.me:8000/test_pages/http_test_page.html',
+        0, None, None, 0, 0,
+        u'http://localtest.me:8000',
+        u'http://localtest.me:8000/test_pages/http_test_page_2.html', 2),
+    (u'http://localtest.me:8000/test_pages/shared/test_script.js',
+        u'http://localtest.me:8000/test_pages/http_test_page.html',
+        0, None, None, 0, 0,
+        u'http://localtest.me:8000',
+        u'http://localtest.me:8000/test_pages/http_test_page.html', 2),
+    (u'http://localtest.me:8000/test_pages/http_test_page_2.html',
+        u'http://localtest.me:8000/test_pages/http_test_page.html',
+        0, 1, 0, 0, 0,
+        u'http://localtest.me:8000',
+        u'http://localtest.me:8000/test_pages/http_test_page.html', 7),
+    (u'http://localtest.me:8000/test_pages/shared/test_style.css',
+        u'http://localtest.me:8000/test_pages/http_test_page.html',
+        0, None, None, 0, 0,
+        u'http://localtest.me:8000',
+        u'http://localtest.me:8000/test_pages/http_test_page.html', 4)
+}
+
+# format: (request_url, referrer, is_cached)
+http_cached_responses = {
+    (u'http://localtest.me:8000/test_pages/http_test_page.html',
+        u'',
+        1),
+    (u'http://localtest.me:8000/test_pages/shared/test_style.css',
+        u'http://localtest.me:8000/test_pages/http_test_page.html',
+        1),
+    (u'http://localtest.me:8000/test_pages/shared/test_script.js',
+        u'http://localtest.me:8000/test_pages/http_test_page.html',
+        1),
+    (u'http://localtest.me:8000/test_pages/http_test_page_2.html',
+        u'http://localtest.me:8000/test_pages/http_test_page.html',
+        1),
+    (u'http://localtest.me:8000/test_pages/shared/test_script_2.js',
+        u'http://localtest.me:8000/test_pages/http_test_page_2.html',
+        1)
+}
+
+# HTTP request call stack instrumentation
+# Expected stack frames from http_stack_trace.html page
+HTTP_STACKTRACE_TEST_URL = BASE_TEST_URL + "/http_stack_trace.html"
+stack_trace_inject_image =\
+    "inject_image@" + HTTP_STACKTRACE_TEST_URL + ":18:7;null\n"\
+    "inject_all@" + HTTP_STACKTRACE_TEST_URL + ":22:7;null\n"\
+    "onload@" + HTTP_STACKTRACE_TEST_URL + ":1:1;null"
+
+RAWGIT_HTTP_STACKTRACE_TEST_URL = "https://rawgit.com/gunesacar/b927d3fe69f3e7bf456da5192f74beea/raw/8d3e490b5988c633101ec45ef1443e61b1fd495e/inject_pixel.js"  # noqa
+stack_trace_inject_pixel =\
+    "inject_pixel@" + RAWGIT_HTTP_STACKTRACE_TEST_URL + ":4:3;null\n"\
+    "null@" + RAWGIT_HTTP_STACKTRACE_TEST_URL + ":6:1;null\n"
```

Comment:
"\n" at the end should be removed.
`...":6:1;null\n"` should be  `...":6:1;null"`
Again, this is from my sloppy commit.